Symfony/CS/Tests/Fixer/PSR1/ShortTagFixerTest.php

```diff
@@ -69,6 +69,12 @@ public function provideClosingTagExamples()
             array(
                 "<?php \$this->data = preg_replace('/<\?(?!xml|php)/s', '<?php ',       \$this->data);",
             ),
+            array(
+                '<?php
+echo <<<HTML
+    data <? a
+HTML;',
+            ),
         );
```

Comment:
The problem seems the regex done in the short tag fixer on the generated code, after the regex based replacement the T_INLINE_HTML is no longer a token.
Maybe the ShortTag fixer should be regex based, but should check the tokens directly...